test/generated/testproto/grpc/import_pb2_grpc.pyi

```diff
@@ -33,15 +33,15 @@ class SimpleServiceServicer(metaclass=abc.ABCMeta):
         context: grpc.ServicerContext,
     ) -> testproto.test_pb2.Simple1:
         """UnaryUnary"""
-        pass
+        ...
 
     @abc.abstractmethod
     def UnaryStream(self,
         request: testproto.test_pb2.Simple1,
         context: grpc.ServicerContext,
     ) -> google.protobuf.empty_pb2.Empty:
         """UnaryStream"""
-        pass
+        ...
```

Comment:
I *think* (not 100% sure) to be conformant, this ellipsis isn't necessary because a docstring was emitted.